docker/test/stateless/run.sh

```diff
@@ -262,14 +265,15 @@ function run_tests()
 
 export -f run_tests
 
+TIMEOUT=$((${MAX_RUN_TIME} - 200))
```

Comment:
Let's put a comment that 200 seconds should be enough to do all the after-tests jobs (gather artifacts)

Reply:
Done